### PR TITLE
{tools}[GCCcore/6.4.0] DB v6.2.32

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/d/DB/DB-6.2.32-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+checksums = ['a9c5e2b004a5777aa03510cfe5cd766a4a3b777713406b02809c17c8e0e7a8fb']
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+sanity_check_paths = {
+    'files': ["include/db.h"],
+    'dirs': [],
+}
```
Comment (vanzod) [inline easybuild/easyconfigs/d/DB/DB-6.2.32-GCCcore-6.4.0.eb]: Why not using the most recent sanity check available in:

https://github.com/easybuilders/easybuild-easyconfigs/blob/b1332a8e969b5f379bb1e613b4e378b7a98d1a3e/easybuild/easyconfigs/d/DB/DB-6.2.32-intel-2017a.eb

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2421.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9e8a36ae76c9d3fc5bd008f4d50601cf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2073.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/37ef29a0ad2008bbe00a38d4cde0f878 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @JackPerdue!

**Pull request merged**